docs/api/wanchain.md

```diff
@@ -1353,14 +1356,15 @@ const mutation = await ledger.revokeAsset(assetId);
 An `asynchronous` class instance `function` which updates ledger data.
 
 ::: warning
-You need `UPDATE_URI_BASE` ledger ability to update ledger's `uriBase` property.
+You need `UPDATE_URI` ledger ability to update ledger's `uriPrefix` and `uriPostifx` properties.
```

Comment:
You need `UPDATE_URI` ledger ability to update ledger's `uriPrefix` and `uriPostfix` properties.